proxy/src/utils/fetchPageData.ts

```diff
@@ -69,8 +69,15 @@ const getProcessedTokens = (tokens: any[]) => {
     const processedToken = { ...token } as any
 
     if (token.type === 'image' && token.text?.includes('|')) {
-      processedToken.text = token.text.split('|')[0]
-      processedToken.aspectRatio = token.text.split('|')[1]
+      const parts = token.href.split(' ')
```

Comment:
Shouldn't this split on a pipe?
```suggestion
      const parts = token.href.split('|')
```